pkg/metrics/metrics.go

```diff
@@ -84,6 +92,8 @@ func init() {
 		failedInstanceUpdateCount,
 		failedInstanceDeleteCount,
 	)
+	// TODO(elmiko) move this registration to an MHC specific metrics package
+	metrics.Registry.MustRegister(MachineHealthCheckNodesCovered)
```

Comment:
Doesn't need to be a separate package, we just need to perform the registration during the controller set up, we can leave this for a second PR though

Reply:
the reason i was thinking about a separate package is that we are running all this in the `init` func, so i thought maybe we would want to be more specific about what gets initialized. i can update this comment to be more specific.